decidim-core/app/services/decidim/action_logger.rb

```diff
@@ -14,7 +14,7 @@ class ActionLogger
     #
     # action - a String representing the name of the action
     # user - the Decidim::User that performed the action
-    # resource - the resource onn which the action was performed
+    # resource - the resource on which the action was performed
```

Comment:
Note that I don't generally fix two-letter words, this is only because it's misspelled as a three-letter word.